### ISSUE TITLE
FEATURE: Sorting options

### ISSUE THREAD

**Issue by dlmdev001** [antrancs/maeve#46]
Sorting options seem to be a necessary option. For example, I'd much prefer to have the albums sorted by artist, so that all my Rolling Stones are with my Rolling Stones. As it is right now, "Awaken, My Love" by Childish Gambino is at the top (due to the quotation marks in the title), followed by "Heroes" by David Bowie, then 'Live' At the Village Vanguard by John Coltrane, which is quite a fuzzy ordering. 

But thank you so much for the gift that this web player is.

**Comment by antrancs** [comment]
@dlmdev001 : Hi, this has been added in the latest build 😃

**Comment by dlmdev001** [comment]
![Screen Shot 2019-07-20 at 7 35 27 PM](https://user-images.githubusercontent.com/50119599/61581996-b3c9d780-ab25-11e9-971f-ebcb99cd1790.png)

You're a wonder. Thank you for this enhancement. I'm a little sad that the large album covers are gone and its merely a list now. I love looking through the albums like in a vinyl collection. 

I've added a screenshot of how it appears on my side. It still seems that they're quite unordered. For example, Bedrich Smetana comes after Bob Dylan, which is not alphabetical. And the Bob Dylan albums are randomly placed.

**Comment by antrancs** [comment]
> You're a wonder. Thank you for this enhancement. I'm a little sad that the large album covers are gone and its merely a list now. I love looking through the albums like in a vinyl collection.

Yeah, that's the trade-off I had to make when implementing this feature. It now looks consistent across the Albums/Playlists/Songs page where you can see thousands of items but still maintaining the scrolling performance. And another difficulty with the cover display is that I couldn't find a way to achieve the grouped sections (A, B, C....) with virtual scrolling as you see with the list display. So I had to go with the current approach. Maybe in the future if I can find a way to do that, I'll try again. But for now, we will stick with the list display :smiling

Regarding the order, it is a temporarily quick and fast implementation to sort the albums based on the artist name. So albums whose artist's name starts with "B" will go into one category. It does not guarantee albums from the same artists will stay together. The reason why the album from Bedrich Smetana  comes after that of Bob Dylan because **"Ma Vlast...."** comes after **"Highway 61..."** and **"Legend"**. But I'm surprised to see the album from Beatenberg (The Hanging...) comes before the other albums. 😄  It must be a bug which I'll look into it.

Right now I just do the first implementation of this feature to get initial feedback. It needs more improvements like albums from the same artist should stay together. I'll keep you posted when it's fixed in future releases.

Thanks for the feedback 👍
